Changes related to the rebranding (#4)
meilisearch/js-project-boilerplate: README.md

<p align="center">

>   <img src="https://github.com/meilisearch/Meilisearch/raw/main/assets/logo.svg" alt="Meilisearch" width="200" height="200" />

Reviewer: Could we use the js here?

```suggestion
  <img src="https://raw.githubusercontent.com/meilisearch/integration-guides/main/assets/logos/meilisearch_js.svg" alt="Meilisearch" width="200" height="200" />
```

</p>

<h1 align="center">JavaScript project boilerplate</h1>

<h4 align="center">

>   <a href="https://github.com/meilisearch/Meilisearch">Meilisearch</a> |

Reviewer: ```suggestion
  <a href="https://github.com/meilisearch/meilisearch">Meilisearch</a> |
```

## 📄 License

> The lincensing file containg the license Meilisearch used accross our projects. The years should be updated accordingly to the current year.

Reviewer: ```suggestion
The licensing file contains the license Meilisearch used across our projects. The years should be updated accordingly to the current year.
```
